### ISSUE TITLE
Missing ScheduledAt params in the SendSmtpEmail object

### ISSUE THREAD

**Issue by antoinebidault** [sendinblue/APIv3-csharp-library#62]
Hi, we are interested in the schedule transactional e-mail features, but the param is missing in the last version of this SDK.
The doc here : https://developers.sendinblue.com/docs/schedule-batch-sendings
It possible to update the Sdk to the last version of the API in order to add to the SendSmtpEmail the additional DateTime param "ScheduledAt". That would save us lots of pain rewriting our own SDK !

**Comment by hieuntp2** [comment]
I have same issue, missing ScheduledAt for .net SKD

**Comment by hieuntp2** [comment]
Hi @antoinebidault , I found a workaround for this issue. The first I need create class inherit SendSmtpEmail look like this:
```
public class MyEmai : SendSmtpEmail
    {
        [DataMember(Name = "scheduledAt")]
        public string ScheduledAt { get; set; }
    }
```

and when you send email, instead pass SendSmtpEmail, we use new class with can set ScheduledAt  value:
```
 apiInstance.SendTransacEmail(new MyEmai()
                {
                    TemplateId = templateId,
                    To = new List<SendSmtpEmailTo>()
                    {
                        new SendSmtpEmailTo("abc@gmail.com")
                    },
                    ScheduledAt = (new DateTime(2022, 12, 22, 4, 52, 00)).ToString("yyyy-MM-dd'T'HH:mm:ss.fffzzz", DateTimeFormatInfo.InvariantInfo)
                });
```

Note, you may want to use date format: "yyyy-MM-dd'T'HH:mm:ss.fffzzz" because API seem required that format

**Comment by amitsendinblue** [comment]
Hi @antoinebidault 
We will add this feature in upcoming release and update here.

**Comment by amitsendinblue** [comment]
Fixed in latest release - v4.0.2 kindly refer the latest version.
